wca-guidelines.md

```diff
@@ -72,7 +72,7 @@ To be more informative, each Guideline is classified using one of the following
 - 3h2++) [CLARIFICATION] "Stickerless" puzzles that significantly differ from most mass-produced "stickerless" puzzles are only permitted at the discretion of the WCA Delegate.
 - 3j+) [REMINDER] In the past, engraved/embossed parts have been permitted. This is no longer permitted.
 - 3j++) [CLARIFICATION] On Clock, pins may be distinguishable from any other pin of the same side.
-- 3j1+) [REMINDER] On Clock, loose pins are not considered reasonable wear, at the discretion of the WCA Delegate.
+- 3j1+) [REMINDER] On Clock, loose pins (i.e. pins that can toggle or recede using gravity instead of being directly pressed)  are not considered reasonable wear.
```

Comment:
Sorry, it looks like I added an extra space here.

Reply:
We need a new PR if we want to fix it, right?

Reply:
Yeah.